.github/workflows/lint.yml

```diff
@@ -21,6 +21,9 @@ jobs:
         with:
           python-version: '2.x'
           architecture: 'x64'
+      - name: Setup Python Installation
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: pip install wheel==0.35.0
```

Comment:
Is there any way for us to ensure the version stays consistent across all installation points?

Reply:
It's a bit ad hoc, but what about a test verifying all 'pip install wheel==X' commands match the current version 0.35.0?

Reply:
The problem with that is that it is not guaranteed for the user to have the same version. As for the possibility of unknown behavior of different wheel behavior, I implemented a test to confirm that the functionality that we need from wheel is still the same so if the version of wheel actually becomes a problem the test will catch it.